### ISSUE TITLE
Umlauts in Adress Lookup?

### ISSUE THREAD

**Issue by fridomac** [clburlison/pinpoint#17]
Hello!

I tried pinpoint (for my munkireport install). Great project, but it seems to have problems when the address returned from google contains an umlaut (at least that´s what I suspect):

```bash
INFO: Read current locationd settings
INFO: Location Services are enabled
DEBUG: Current location domain of client.plist is: 
DEBUG: {
    Authorized = 1;
    BundleId = "com.apple.locationd.executable-/System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python";
    Executable = "/System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python";
    Hide = 0;
    Registered = "/System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python";
    Requirement = "cdhash H\"63f2adf4dac0829622cee884703685c693d4adfb\" or cdhash H\"7f059a96eff2890f320cfc9b4f9ee7f3ee664571\"";
    Whitelisted = 0;
}
INFO: Python is enabled
DEBUG: Updating 'LastCheckDate' timestamp
DEBUG: Storing past location
DEBUG: Wireless interface is currently: On
DEBUG: Running lookup loop: 0
INFO: Process a lookup request
DEBUG: Successful lookup request: 49.4488988417, 11.0870083546
INFO: Process a lookup request
DEBUG: Successful lookup request: 49.4488988417, 11.0870083546
INFO: Process a lookup request
DEBUG: Successful lookup request: 49.4488673252, 11.0870089456
INFO: Process a lookup request
DEBUG: Successful lookup request: 49.4488676288, 11.0870091044
INFO: Process a lookup request
DEBUG: Successful lookup request: 49.4488680774, 11.0870093973
INFO: Process a lookup request
DEBUG: Successful lookup request: 49.4488681514, 11.0870094331
INFO: Download json address file from Google
INFO: Download successful
Traceback (most recent call last):
  File "/Library/Application Support/pinpoint/bin/pinpoint", line 579, in <module>
    main()
  File "/Library/Application Support/pinpoint/bin/pinpoint", line 565, in main
    address_lookup('google')
  File "/Library/Application Support/pinpoint/bin/pinpoint", line 506, in address_lookup
    add = dict(Address=str(address),)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xdf' in position 7: ordinal not in range(128)
```

Bye, Frido.

**Comment by clburlison** [comment]
Hey Frido,

Would you mind testing the latest pre-release? The issue was resolved over here.

https://github.com/clburlison/pinpoint/releases/tag/v0.0.3.1

**Comment by fridomac** [comment]
Will test tomorrow morning when I return to the office and report back, thank you for fixing this!

Bye, Frido.

**Comment by fridomac** [comment]
Test was successful, it worked and reported the location (did only get a grey square instead of the map in munkireport, but the data on the left was OK).
Thanks again for fixing it!

Bye, Frido.

**Comment by fridomac** [comment]
Ok, forget the grey map issue, I updated to the latest munkireport-wip, then it started working.

**Comment by clburlison** [comment]
Thanks again for reporting and testing! 

The issue you saw with the Map being grey is actually due to a timeout on the connection when attempting to pull the map data from Google. A refresh of the page normally resolves the issue for me.

**Comment by clburlison** [comment]
I'm going to go ahead and re-open this issue as the solution I wrote was less than ideal. Just because it worked doesn't mean it was [good practice](http://stackoverflow.com/questions/3828723/why-should-we-not-use-sys-setdefaultencodingutf-8-in-a-py-script/3828742#3828742).

Ideally, I need to revisit this and explicitly encode the address string.

Reference: https://macadmins.slack.com/archives/python/p1457377263002622

**Comment by clburlison** [comment]
Fixed correctly as of c882c29
